### ISSUE TITLE
Unable to load Authentication\/AUTHENTICATION

### ISSUE THREAD

**Issue by De1Facto** [sipcapture/homer#159]
Hi, after installing homer (pulled the source yesterday), and finally getting the api to work with nginx I can't login with default admin/test123. 
When navigating to X.X.X.X/api/v1/session  I see the following:

```
{
    "status": 500,
    "error": "Exception",
    "message": "Unable to load Authentication\/AUTHENTICATION."
}
```

When navigating to X.X.X.X/api/v1/test

```Your api call works! Well done!```

output of uname -a : ` 3.10.0-327.13.1.el7.x86_64`

cat /etc/php-fpm.d/www.conf

[www.conf](https://gist.github.com/De1Facto/fe0bc6a7e12c77d77bbb946ac824cf49)

cat /etc/nginx/nginx.conf
[nginx.conf](https://gist.github.com/De1Facto/4da8d7aa25a4c7e5445b7a184adff972)

**Comment by adubovikov** [comment]
Hi,

please be sure that you have installed php5-pdo

Wbr,
Alexandr

**Comment by De1Facto** [comment]
Hi, 

Thank you for the fast response. I do have php-pdo installed:
```# rpm -qa | grep php-pdo
php-pdo-5.4.16-36.el7_1.x86_64 ```

**Comment by lmangani** [comment]
Anything in the logs as this event hits?

**Comment by adubovikov** [comment]
please also provide php and OS version,

**Comment by De1Facto** [comment]
OS version:
CentOS Linux release 7.2.1511 (Core)
PHP version:
PHP 5.4.16

**Comment by De1Facto** [comment]
I think I found the problem since I saw in php-fpm error.log the following:

[11-Apr-2016 08:04:35 UTC] PHP Warning:  include_once(preferences.php): failed to open stream: No such file or directory in /var/www/homer/api/configuration.php on line 28

so it appears that I didn't rename the preferences_example.php script as preferences.php. 
When I did it I was able to login. Thank you very much for your assistance.
